src/settings/connectionsettings.ts

```diff
@@ -370,6 +375,11 @@ export function isSonarQubeConnection(connection: BaseConnection): connection is
   return (connection as SonarQubeConnection).serverUrl !== undefined;
 }
 
+export function getSonarQubeCloudTokenStorageKey(organizationKey: string, region?: SonarCloudRegion): string {
+  const regionPrefix = region ? `${region}_` : '';
+  return regionPrefix + organizationKey;
+}
+
```

Comment:
This looks a lot like a re-implementation of `getTokenStorageKey` below, could we imagine to refactor this a bit?